components/object_store/src/aliyun.rs

```diff
@@ -112,11 +121,23 @@ impl ObjectStore for AliyunOSS {
         Ok(GetResult::Stream(stream::once(async { Ok(bytes) }).boxed()))
     }
 
-    async fn get_range(&self, _location: &Path, _range: Range<usize>) -> Result<Bytes> {
-        Err(Error::Unimplemented {
-            op: "get_range".to_string(),
+    async fn get_range(&self, location: &Path, range: Range<usize>) -> Result<Bytes> {
```

Comment:
There is a same name method in `cache.rs`, it currently invoke `get` to fetch data, after this PR, it can invoke `get_range` directly.

Reply:
Let me fix it in this pr.

Reply:
I find that `CachedStore` has to cache the whole file, even if only a range of file fetched. I will solve this in #392.

Reply:
> has to cache the whole file

Any explain for this? I don't remember it has this requirement.

Reply:
Actually, the granularity of the `CachedStore` is file level, and the code about the `get_range` of `Cache` is:
```rust
    async fn get_range(&self, location: &Path, range: Range<usize>) -> Result<Bytes> {
        if self.state.lock().await.contains(location) {
            self.local_store.get_range(location, range).await
        } else {
            let remote_obj = self.remote_store.get(location).await?;
            let bytes = remote_obj.bytes().await?;
            let _ = self.try_put_local(location, bytes).await?;
            self.local_store.get_range(location, range).await
        }
    }
```